scripts/merge-and-autograde.sh

```diff
@@ -110,37 +137,16 @@ do
 		if [ -e "../repos/$gitHandle-CS4100/PA${pa_number}/${fileName}" ]
 		then
 			yes | cp -rf "../repos/$gitHandle-CS4100/PA${pa_number}/${fileName}" "tmp/${zipName}/"
+			run_grader_and_generate_report
 		elif [ -e "../repos/$gitHandle-CS4100/pa${pa_number}/${fileName}" ]
 		then
 			yes | cp -rf "../repos/$gitHandle-CS4100/pa${pa_number}/${fileName}" "tmp/${zipName}/"
+			run_grader_and_generate_report		
```

Comment:
This is wrong. You are running the grader for each file being copied while you should only run the grader once after all the files have been copied for the assignment.
E.g.:
Suppose files A and B are needed for assignment 1.

Ideal:
You copy A and B to tmp, then run autograder.

What your PR is doing:
copy A, run autograder, copy B, run autograder.

Please fix the PR.